bpf/lib/trace.h

```diff
@@ -45,15 +45,14 @@ enum {
 };
 
 /* Reasons for forwarding a packet. */
-enum {
+enum trace_reason {
 	TRACE_REASON_POLICY = CT_NEW,
 	TRACE_REASON_CT_ESTABLISHED = CT_ESTABLISHED,
 	TRACE_REASON_CT_REPLY = CT_REPLY,
 	TRACE_REASON_CT_RELATED = CT_RELATED,
 	TRACE_REASON_CT_REOPENED = CT_REOPENED,
-};
-
-#define TRACE_REASON_ENCRYPTED	    0x80
+	TRACE_REASON_ENCRYPTED = 0x80
+} __packed;
```

Comment:
Note: If we also named the enum for `CT_NEW` & co. (e.g., to `ct_status`), then we could use Coccinelle to detect those errors in the future. We'd just have to write another SmPL script that requires the second-to-last `send_trace_notify` argument to be either of type `enum ct_status` or to be `TRACE_REASON_ENCRYPTED`.